src/term-spec.js

```diff
@@ -32,7 +32,7 @@ declare export class NamedObjectProperty extends ObjectProperty {
   name: PropertyName;
 }
 declare export class MethodDefinition extends NamedObjectProperty {
-  body: FunctionBody;
+  body?: FunctionBody; // undefined only on abstract class methods
```

Comment:
I [did this and while at it I found two bugs which I tried to fix](https://github.com/sweet-js/sweet-spec/pull/4/commits/1681ece54ba44d4770beb7539e37954d10da2e64). Note that I also had to add `AbstractMethod`, `AbstractGetter` and `AbstractSetter` which mirror the non-abstract counterparts.